source/plot/LinePlotData2D.ooc

```diff
@@ -12,30 +12,42 @@ LineStyle: enum {
 
 LinePlotData2D: class extends PlotData2D {
 	lineStyle: LineStyle { get set }
+	dataSeries: VectorList<FloatPoint2D> { get set }
```

Comment:
I don't like it, but we do have a `FloatPoint2DVectorList`. Any reason for not using it?